src/test/resources/fixtures/hmh-ri/caliperEventAssessmentStarted.json

```diff
@@ -0,0 +1,46 @@
+{
+  "@context":"http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "type":"AssessmentEvent",
+  "id":"urn:uuid:27734504-068d-4596-861c-2315be33a2a2",
+  "actor":{
+    "id":"urn:uuid:0f4fedbe-2227-415f-8553-40731a627171",
+    "type":"Person",
+    "name" : "Casandra Rath"
+  },
+  "action":"Started",
+  "object":{
+    "id":"urn:uuid:c050e852-5edb-4743-92d1-b53466de3a5f", // assignment activity id
+    "type":"Assessment",
+    "name":"ReadingInventory",
+    "maxAttempts":1,
+    "maxSubmits":1,
+    "maxScore":25.0,
+    "version":"1.0"
+  },
+  "generated":{
+    "id":"urn:uuid:c35603f7a3434e2cb74ef14d60e70f42", // ri-quiz id
```

Comment:
Hi @krisiye, the comment here says ri-quiz id - but I don't think that's correct, this id is an attempt id